Azure/sonic-buildimage
# [multi-asic][vs]: Remove unused backend asic interfaces

#### Why I did it
In msft_four_asic_vs hwsku, there are 2 front-end asics and 2 back-end asics.
The VS DUT has 8 front panel ports, each asic was assigned 8 interfaces.
For the backend asics, only 4 interfaces are used to connect to front-end asics.
As 4 interfaces of each backend asic was never used used internally and are not present in minigraph, it can be removed.
Also, as these interfaces are not connected, mtu is not assigned to these interfaces while generating configuration, which causes snmp interfaces test case to fail.

#### How I did it
Remove 4 unused interfaces from backend asics' portconfig.ini and lanemap.ini.

#### How to verify it
Bring up 4 asic VS. ./testbed-cli.sh -t vtestbed.csv -m veos_vtb -k ceos add-topo vms-kvm-four-asic-t1-lag password.txt
Load minigraph: ./testbed-cli.sh -t vtestbed.csv -m veos_vtb deploy-mg vms-kvm-four-asic-t1-lag lab password.txt
Ensure all interfaces and BGP sessions are coming up.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
